spec/ui_spec.rb

```diff
@@ -79,6 +79,45 @@
     expect(ui.ask_for_notes).to eq("notes")
   end
 
+  it "prints the prompt message for repeating search" do
+    input.string = "\n"
+    ui.ask_search_again
+    expect_msg(UI::SEARCH_AGAIN)
```

Comment:
What does `msg` mean? Abbreviations like that do not help readability and clarity of tests.